src/main/java/org/killbill/billing/plugin/stripe/StripePaymentPluginApi.java

```diff
@@ -204,7 +204,9 @@ else if (stripeConfigPropertiesConfigurationHandler.getConfigurable(context.getT
                         if (charge.getPaymentIntent() != null) {
                             final PaymentIntent intent = PaymentIntent.retrieve(charge.getPaymentIntent(), requestOptions);
                             logger.info("Fixing Stripe transaction {}", intent.getId());
-                            dao.updateResponse(transaction.getKbTransactionPaymentId(), intent, context.getTenantId());
+                            // FIXME-STRIPE-87: should we use searchParams.toMap() instead?
```

Comment:
I don't think so, otherwise `lastCharge` ends up being `charge` (single charge for a given `kbTransactionId`). We could be in a Janitor flow here where we're fixing an older charge (`charge` is older then `lastCharge`).